simulator/src/combinationalAnalysis.js

```diff
@@ -240,6 +243,15 @@ function drawCombinationalAnalysis(combinationalData, inputList, outputListNames
     var startPosY = 200;
 
     var currentPosY = 300;
+    if (simulationArea.maxWidth && simulationArea.maxHeight) {
+        if (simulationArea.maxHeight + 300 > simulationArea.maxWidth) {
```

Comment:
Hey @pulkit-30 Can you explain why the height and width relation like this is required?

Reply:
Also I would suggest putting the `300` into some variable with an appropriate name to avoid magic numbers

Reply:
> Hey @pulkit-30 Can you explain why the height and width relation like this is required?

Yep! 
small logic to prevent growth in a single dimension

ref: https://github.com/CircuitVerse/CircuitVerse/pull/2205#issuecomment-823231798

Reply:
Oh understood. Can you just make the `300` into a variable. Then we're good to go

Reply:
Done 🚀